### PR TITLE
Star Alliance's blaster cards

### PR DESCRIPTION
**Pull request by romanogit**

### CODE REVIEW

File: server/game/cards/03-WC/BlasterCard.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const Card = require('../../Card.js');
+
+class BlasterCard extends Card {
+    getAttachedAbility(ability, blasterName, officerName) {
+        let choices = {
+            'Deal 2 damages': () => true
```
Comment (cryogen) [inline server/game/cards/03-WC/BlasterCard.js]: No 's'

File: server/game/cards/03-WC/BlasterCard.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        };
+
+        choices[`Move ${blasterName}`] = () => true;
```
Comment (cryogen) [inline server/game/cards/03-WC/BlasterCard.js]: Why not include this in the above initialiser?

File: server/game/cards/03-WC/BlasterCard.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            optional: true,
+            targets: {
+                action: {
+                    mode: 'select',
+                    choices: choices
+                },
+                creature: {
+                    dependsOn: 'action',
+                    cardType: 'creature',
+                    cardCondition: (card, context) => (context.selects.action.choice === 'Deal 2 damages') ? true : card.name === officerName && !card.upgrades.includes(this),
```
Comment (cryogen) [inline server/game/cards/03-WC/BlasterCard.js]: No 's' on damage

File: server/game/cards/03-WC/BlasterCard.js (hunk 2 above)
Comment (romanogit) [reply server/game/cards/03-WC/BlasterCard.js]: I don't think we can have an expression as a json key. I get an error. That's why I used brackets.

**Comment by romanogit** [comment]
This PR is now complete and merged.

**Pull request merged**